test/test_repo_template/asv_test_repo/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 from __future__ import absolute_import, division, unicode_literals, print_function
 
-dummy_value = {dummy_value}
+dummy_value = {dummy_value} # noqa 
```

Comment:
For a `noqa` we should have the error code you want to ignore, not all them. And also a comment explaining why we want flake8 to ignore the problem here, instead of fixing the error.

Reply:
Thanks, Marc, Let me add this asap to be more descriptive